roles/ovirt-cluster-upgrade/tasks/main.yml

```diff
@@ -83,14 +78,11 @@
       with_items:
         - "{{ stopped_vms | default([]) }}"
   
-    - include: pinned_vms.yml
-  
     # Update only those hosts that aren't in list of hosts were VMs are pinned
     # or if stop_non_migratable_vms is enabled, which means we stop pinned VMs
```

Comment:
Hmm, this comment is not valid anymore, right?

Reply:
will fix